pkgs/applications/networking/remote/waypipe/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchFromGitLab
+, meson, ninja, pkgconfig, scdoc
+, wayland, wayland-protocols, openssh
+, mesa, lz4, zstd, ffmpeg_4, libva
+}:
+
+stdenv.mkDerivation rec {
+  pname = "waypipe-unstable";
+  version = "0.6.1";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "mstoeckl";
+    repo = pname;
```

Comment:
```suggestion
    repo = "waypipe";
```

Reply:
Will this be an improvement? IMO this is subjective (or do we have guidelines regarding this?) and AFAIK we currently mix both variants in Nixpkgs. I'm fine with either version but tbh it feels a bit like bikeshedding.

Reply:
I mean, pname is set to "waypipe-unstable". So we should use `repo = "waypipe"` for now.

Reply:
Oh, sorry! My bad, I've completely missed that and was surprised about the change :o :D

Thanks for paying better attention!

(Note: I'll keep `unstable` in `pname` for now as this is afaik the preferred solution, see https://github.com/NixOS/nixpkgs/pull/67460#discussion_r317415142)